src/systemd-sonic-generator/systemd-sonic-generator.c

```diff
@@ -462,6 +609,13 @@ static int create_symlink(char* unit, char* target, char* install_dir, int insta
         }
     }
 
+    if (is_devnull(dest_path)) {
+        if (remove(dest_path) == -1) {
```

Comment:
[minor]
It would be better to check if the return code is equal to zero:
`remove(dest_path) != 0`
I understand that the `remove` can return only `-1` in case of an error. But, it looks "safer"

Reply:
done